package.json

```diff
@@ -66,6 +66,7 @@
 		"cypress": "^9.6.0",
 		"cypress-jest-adapter": "^0.1.1",
 		"eslint": "^8.3.0",
+		"eslint-plugin-deprecation": "^1.3.2",
```

Comment:
Can we add this package to the `tools/eslint-config-web3-base/package.json`? May be other eslint plugins as well which are used in from that package.

Reply:
Added ( + `eslint-plugin-header`) and removed from workspace root.